clients/upsclient.c

```diff
@@ -315,22 +310,35 @@ int upscli_init(int certverify, const char *certpath,
 	}
 	
 #ifdef WITH_OPENSSL
+
+	SSL_load_error_strings();
```

Comment:
This is done automatically, and was deprecated, starting with openssl 1.1.0; my suggestion would be to move it inside `OPENSSL_VERSION_NUMBER < 0x10100000L`